src/funnels/uniV3/TickMath.sol

```diff
@@ -30,63 +30,65 @@ library TickMath {
         pure
         returns (uint160 sqrtPriceX96)
     {
-        uint256 absTick =
-            tick < 0 ? uint256(-int256(tick)) : uint256(int256(tick));
+        unchecked {
```

Comment:
We could add a comment at the top that "this file has a minor solidity version upgrade and makes use of  `unchecked` math (as in the original Uniswap file), compared to the Gelato version".

Reply:
Not sure it's strictly necessary, we already link to both the gelato and uniswap version, so it's pretty easy for anyone to make the diff.